app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
 
     defaultConfig {
         applicationId "io.transcend.samplesdk"
-        minSdk 24
+        minSdk 21
```

Comment:
How did minSdk go down?

Reply:
Athlectic wanted minSDK>=21 so, we have updated our library to be compatable.